js/src/react/components/TabContent/TabContent.js

```diff
@@ -9,7 +9,7 @@ export default function TabContent({sectionId, isActive, tabComponent}) {
 
     useEffect(() => {
         if (isActive && !sectionData) {
-            if ( sectionId === 'firewall' ) {
+            if ( sectionId === 'firewall' || sectionId === 'critical_updates' ) {
```

Comment:
Please, remove the condition `sectionId === 'firewall'` - there is no such tab/section. This was just for an example.